src/XMakeCommandLine/UnitTests/XMake_Tests.cs

```diff
@@ -633,7 +632,8 @@ public void ConfigurationInvalid()
                     "</Project>";
                 File.WriteAllText(pathToProjectFile, projectString);
 
-                output = RunProcessAndGetOutput(newPathToMSBuildExe, "\"" + pathToProjectFile + "\"", expectSuccess: false);
+                var msbuildParameters = "\"" + pathToProjectFile + "\"";
+                output = runMSBuildWithinCoreRun(newPathToMSBuildExe, msbuildParameters, expectSuccess: false);
```

Comment:
Is this unconditionally running with corerun?  It should be conditional on whether we're built for coreclr--this branch should still work on full-framework (I know it doesn't currently).